setup.py

```diff
@@ -40,7 +40,7 @@
         "requests >= 2.21.0, < 3.0.0",
         "python-levenshtein >= 0.12.0, < 1.0.0",
         "normality >= 2.4.0, < 3.0.0",
-        "sqlalchemy >= 1.2.0, < 3.0.0",
+        "sqlalchemy >= 1.3.2, < 3.0.0",
```

Comment:
I didn't increase it yet to `1.4` because I still have the lingering issue of tests in `dataset` failing for `SQLAlchemy` version `1.4.49`. Not knowing the reason behind it yet, I didn't want to break these libs for anyone using them in a way that triggers this issue.

Reply:
I updated the minimum `SQLAlchemy` version. I think you're right, this might affect anyone running FTM on its own, or in conjuncture with other scripts. I re-build `memorious` and it builds fine, I re-ran a `memorious` test and it works.